### PR TITLE
Consolidate conftest.py approach

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description
- Consolidate conftest.py creation approach aligned with A.
- Re-placed fixtures and deleted their `conftest.py` if only used within single file

In general our approach is to keep fixtures close to where they are being used:
- Add the test fixtures directly into the relevant `test_XXX.py` file if only used there
- If shared across several test files, add the fixtures in a `conftest.py` placed in the sub-directory
- Move one level up if the fixtures are shared across sub-directories etc.

## Checklist

- [x] I have not referenced individuals, products or companies in any commits, directly or indirectly
- [x] I have not added data or restricted code in any commits, directly or indirectly
- [x] I have updated the docstring of any public function/class/model changed
- [x] I have added the PR number to the change description in the changelog fragment, e.g. `Enable feature XXX (#123)` (if applicable)

## Types of changes

- [x] Docs/refactoring (non-breaking change which improves codebase)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by petar-qb — APPROVED** [review]
Now everything is in its place. I double checked the tests/unit/vizro/actions folder and it already follows the practices from the the PR description.

File: vizro-core/tests/unit/vizro/conftest.py [hunk 1]
```diff
@@ -25,20 +23,6 @@ def standard_px_chart(gapminder):
     )
 
 
-@pytest.fixture
```
Comment (maxschulz-COL) [inline vizro-core/tests/unit/vizro/conftest.py:28]: My only concern is, will these very standard fixtures be "found"?

**Review by maxschulz-COL — APPROVED** [review]
LGTM! Just have the concern that we will duplicate many fixtures if we put even the most general fixtures into files if they are currently only used once. But probably this is still the cleaner approach, so good to go!

File: vizro-core/tests/unit/vizro/conftest.py (hunk 1 above)
Comment (huong-li-nguyen) [reply vizro-core/tests/unit/vizro/conftest.py:28]: Currently, there are not that many fixtures and we managed to re-use many after reminding people during PRs 😄 

I think we just need to double-check every now and then and consolidate

**Pull request merged**